### PR TITLE
Api Diff between preview 7 and preview 6

### PR DESCRIPTION
**Pull request by Anipik**
cc  @carlossanlop @jozkee @dotnet/ncl @steveharter @AaronRobinsonMSFT @jkoritzinsky @tannergooding @CarolEidt @echesakovMSFT @layomia @krwq @tarekgh @bartonjs @stephentoub  @tommcdon @krwq @ericstj @richlander 

@mkArtakMSFT @pranavkm @halter73 @Tratcher @BrennanConroy @anurse

### CODE REVIEW

**Comment by AaronRobinsonMSFT** [comment]
Signed off on `System.Runtime.InteropServices.md`

**Comment by tommcdon** [comment]
adding @MichalStrehovsky for DynamicallyAccessedMemberTypes enum rename

File: release-notes/5.0/preview/api-diff/preview7/Asp.Net/5.0-preview7_Microsoft.AspNetCore.Authentication.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Microsoft.AspNetCore.Authentication
+
+``` diff
+ namespace Microsoft.AspNetCore.Authentication {
```
Comment (pranavkm) [inline release-notes/5.0/preview/api-diff/preview7/Asp.Net/5.0-preview7_Microsoft.AspNetCore.Authentication.md:4]: /cc @haok

Comment (HaoK) [reply release-notes/5.0/preview/api-diff/preview7/Asp.Net/5.0-preview7_Microsoft.AspNetCore.Authentication.md:4]: Yep these changes look good

**Comment by MichalStrehovsky** [comment]
DynamicallyAccessedMemberTypes enum rename looks good.

**Comment by anurse** [comment]
FYI @Anipik I'm no longer with the .NET team, so should be removed from the list of reviewers for API diffs. The others you currently have on the list should already be providing coverage for my areas (or reach out to @Pilchie for other area owners in ASP.NET).

Looks like y'all have done a lot of cool work though! 🎉

**Comment by Anipik** [comment]
can everyone tagged  please review their areas ?

**Comment by tarekgh** [comment]
System.Text and System.Threading.Tasks looks good to me.

File: release-notes/5.0/preview/api-diff/preview7/.Net/5.0-preview7_System.Net.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# System.Net
+
+``` diff
+ namespace System.Net {
+     public class WebHeaderCollection : NameValueCollection, IEnumerable, ISerializable {
+-        public new string this[string name] { get; set; }
```
Comment (karelz) [inline release-notes/5.0/preview/api-diff/preview7/.Net/5.0-preview7_System.Net.md:6]: This was removed from ref in PR #37949 ... we still inherit it from [`NameValueCollection`](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Collections.Specialized/src/System/Collections/Specialized/NameValueCollection.cs#L334-L349)

Approved on behalf of @dotnet/ncl

File: release-notes/5.0/preview/api-diff/preview7/Asp.Net/5.0-preview7_Microsoft.AspNetCore.Server.Kestrel.Core.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Microsoft.AspNetCore.Server.Kestrel.Core
+
+``` diff
+ namespace Microsoft.AspNetCore.Server.Kestrel.Core {
```
Comment (Tratcher) [inline release-notes/5.0/preview/api-diff/preview7/Asp.Net/5.0-preview7_Microsoft.AspNetCore.Server.Kestrel.Core.md:4]: :shipit:

File: release-notes/5.0/preview/api-diff/preview7/Asp.Net/5.0-preview7_Microsoft.Net.Http.Headers.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# Microsoft.Net.Http.Headers
+
+``` diff
+ namespace Microsoft.Net.Http.Headers {
+     public class SetCookieHeaderValue {
++        public IList<StringSegment> Extensions { get; }
```
Comment (Tratcher) [inline release-notes/5.0/preview/api-diff/preview7/Asp.Net/5.0-preview7_Microsoft.Net.Http.Headers.md:6]: :shipit:

**Pull request merged**